oap-server/server-starter/src/main/resources/component-libraries.yml

```diff
@@ -254,6 +254,9 @@ pulsar-consumer:
 Ehcache:
   id: 75
   languages: Java
+servicecomb-java-chassis:
+  id: 76
```

Comment:
id=28 ServiceComb is chassis, when the ServiceComb community added the plugin.

Reply:
Edit